library/almalinux

```diff
@@ -1,16 +1,26 @@
 Maintainers: The AlmaLinux OS Foundation <cloud-infra@almalinux.org> (@AlmaLinux)
 GitRepo: https://github.com/AlmaLinux/docker-images.git
 
-Tags: latest, 8, 8.5, 8.5-20211112
-GitFetch: refs/heads/almalinux-8-x86_64-default
-GitCommit: 6e2df1f70f5c144198756f4be9a30e412e8bb065
-arm64v8-GitFetch: refs/heads/almalinux-8-aarch64-default
-arm64v8-GitCommit: 445a8085003f2b1d2de35218d49bf12944497575
-Architectures: amd64, arm64v8
+Tags: latest, 8, 8.5, 8.5-20220306
+GitFetch: refs/heads/al-8.5.4-20220306
+GitCommit: 800ecbd636ec5ebbedc99e46391dbea6e71c2646
+File: Dockerfile-x86_64-default
+arm64v8-GitFetch: refs/heads/al-8.5.4-20220306
+arm64v8-GitCommit: 800ecbd636ec5ebbedc99e46391dbea6e71c2646
+arm64v8-File: Dockerfile-aarch64-default
+ppc64le-GitFetch: refs/heads/al-8.5.4-20220306
+ppc64le-GitCommit: 800ecbd636ec5ebbedc99e46391dbea6e71c2646
+ppc64le-File: Dockerfile-ppc64le-default
```

Comment:
If handling these as a single commit is easier for you or your process, that's totally fine, but just FYI it's a tiny bit more load (since now every arch has to download the tarballs for every other arch to build).  I'm not concerned about that, but figured it's worth noting.

You can also optimize your own overhead updating this file a little bit now by removing the per-arch `GitFetch`/`GitCommit` values (since they're all the same):

```suggestion
GitFetch: refs/heads/al-8.5.4-20220306
GitCommit: 800ecbd636ec5ebbedc99e46391dbea6e71c2646
amd64-File: Dockerfile-x86_64-default
arm64v8-File: Dockerfile-aarch64-default
ppc64le-File: Dockerfile-ppc64le-default
```

(Also, if you let `File:` stay defaulted to `Dockerfile` which doesn't exist, then you have less chance of accidentally forgetting an `arch-File` value in the future. :see_no_evil:)

Reply:
@tianon Thank you for your input and for accepting a single branch approach for each release. Your suggestions certainly help to keep the file simple.